src/Lykke.Service.Assets/Cache/AssetsForClientCacheManager.cs

```diff
@@ -32,144 +37,109 @@ public class AssetsForClientCacheManager : IAssetsForClientCacheManager
             }
         }
 
-        public async Task SaveAssetForClient(string clientId, bool isIosDevice, IEnumerable<string> clientAssetIds)
-        {
-            try
-            {
-                var key = GetKeyAvailableAssets(clientId, isIosDevice);
-                var text = clientAssetIds.ToList().ToJson();
-                await _cache.SetStringAsync(key, text, new DistributedCacheEntryOptions()
-                {
-                    AbsoluteExpirationRelativeToNow = _settings.AssetsForClientCacheTimeSpan
-                });
-            }
-            catch (Exception ex)
-            {
-                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(SaveAssetForClient), clientId, ex);
-            }
-        }
+        public async Task SaveAssetForClientAsync(string clientId, bool isIosDevice, IEnumerable<string> clientAssetIds)
+            => await SetAsync(GetKeyAvailableAssets(clientId, isIosDevice), clientId, clientAssetIds.ToList());
 
-        public async Task SaveCashInViaBankCardEnabledForClient(string clientId, bool isIosDevice, bool cashInViaBankCardEnabled)
-        {
-            try
-            {
-                await _cache.SetStringAsync(GetKeyCashInViaBankCardEnabled(clientId, isIosDevice), cashInViaBankCardEnabled.ToJson(),
-                    new DistributedCacheEntryOptions() { AbsoluteExpirationRelativeToNow = _settings.AssetsForClientCacheTimeSpan });
-            }
-            catch (Exception ex)
-            {
-                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(SaveCashInViaBankCardEnabledForClient), clientId, ex);
-            }
-        }
+        public async Task SaveCashInViaBankCardEnabledForClientAsync(string clientId, bool isIosDevice, bool enabled)
+            => await SetAsync(GetKeyCashInViaBankCardEnabled(clientId, isIosDevice), clientId, enabled);
 
-        public async Task SaveSwiftDepositEnabledForClient(string clientId, bool isIosDevice, bool swiftDepositEnabled)
-        {
-            try
-            {
-                await _cache.SetStringAsync(GetKeySwiftDepositEnabled(clientId, isIosDevice), swiftDepositEnabled.ToJson(),
-                    new DistributedCacheEntryOptions() { AbsoluteExpirationRelativeToNow = _settings.AssetsForClientCacheTimeSpan });
-            }
-            catch (Exception ex)
-            {
-                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(SaveSwiftDepositEnabledForClient), clientId, ex);
-            }
-        }
+        public async Task SaveSwiftDepositEnabledForClientAsync(string clientId, bool isIosDevice, bool enabled)
+            => await SetAsync(GetKeySwiftDepositEnabled(clientId, isIosDevice), clientId, enabled);
+
+        public async Task SaveAssetConditionsForClientAsync(string clientId, IList<IAssetCondition> conditions)
```

Comment:
async await words are redundant here. Just return method's result

Reply:
I guess it's not a problem here. But without async await it will look inconsistent.